beacon-chain/rpc/eventsv1/events.go

```diff
@@ -94,7 +101,7 @@ func (s *Server) handleBlockEvents(
 		}
 		v1Data, err := migration.BlockIfaceToV1Blockheader(blkData.SignedBlock)
 		if err != nil {
-			return err
+			return status.Errorf(codes.Internal, "could not create data required: %v", err)
```

Comment:
It only makes sense to return `status.Error` from top-level functions.
```suggestion
			return err
```